src/components/FieldPreview/FieldPreview.js

```diff
@@ -0,0 +1,87 @@
+// @flow strict
+
+import React from 'react'
+import { t } from 'ttag'
+
+import {
+  JLink,
+  JIcon,
+} from 'components/base'
+import { clipboard } from 'services'
+
+import style from './fieldPreview.m.scss'
+
+type Props = {
+  +label: string,
+  +body: string,
+  link?: string,
+  contact?: string,
+  copy?: string,
+}
+
+function copyToClipboard({ currentTarget }: SyntheticEvent<HTMLButtonElement>): void {
+  const value = currentTarget.getAttribute('data-value')
+
+  if (value) {
+    clipboard.copyText(value)
+  }
+}
+
+function joinAddContact(body: string): string {
+  return body
+    ? `/contacts/add/address=${body}`
```

Comment:
Error: `/contacts/add?address=${body}`

If there is no body, better to leave link to `/contacts/add` than empty link